src/styles/icons/_di.scss

```diff
@@ -1,110 +1,17 @@
-@font-face {
-  font-family: "DSO IconFont";
-  font-style: normal;
-  font-weight: normal;
-  src: url("#{$dso-icon-font-path}DSO-IconFont.eot"); // sass-lint:disable-line no-duplicate-properties
-  src: url("#{$dso-icon-font-path}DSO-IconFont.woff") format("woff"), // sass-lint:disable-line no-duplicate-properties
-  url("#{$dso-icon-font-path}DSO-IconFont.ttf") format("truetype"),
-  url("#{$dso-icon-font-path}DSO-IconFont.svg#DSO-IconFont") format("svg");
+@mixin di($icon, $size: 16px) {
+  background: url("/dso-icons.svg#img-#{$icon}") no-repeat;
+  background-position: center;
+  background-size: cover;
+  height: $size;
+  width: $size;
 }
 
-$di-icon-map: (
-  "cross": "\0041",
-  "info": "\0042",
-  "check": "\0043",
-  "warning": "\0044",
-  "forbidden": "\0045"
-);
-
-@mixin di-font($pseudo: "before") {
-  &::#{$pseudo} {
-    -moz-osx-font-smoothing: grayscale; // sass-lint:disable-line no-vendor-prefixes
-    -webkit-font-smoothing: antialiased; // sass-lint:disable-line no-vendor-prefixes
-    display: inline-block;
-    font-family: "DSO IconFont";
-    font-style: normal;
-    font-weight: normal;
-    line-height: 1;
-  }
+@mixin di-variant($icon) {
+  background-image: url("/dso-icons.svg#img-#{$icon}");
 }
 
-@mixin di($icon, $pseudo: "before") {
-  &::#{$pseudo} {
-    content: map-get($di-icon-map, $icon);
-  }
+svg.di {
+  height: 1em;
```

Comment:
deze `1em` is eigenlijk de `$default-img-size`, die dan ook elders kan worden gezet.